services/parser-database/connector.py

```diff
@@ -20,20 +21,23 @@ def get_documents_to_parse():
 
 
 def get_keywords(text):
-    """Get keywords that relate to this article
+    """Get keywords that relate to this article (Javier's service)
 
     Args:
         text (sting): text to extract keywords from
 
     Returns:
         [list]: list of extracted keywords
     """
-    splited_text = text.split()
```

Comment:
[nit] the past tense of split is actually just split, so this can just be "split_text"